_rules/first-focusable-element-is-link-to-main-8a213c.md

```diff
@@ -0,0 +1,728 @@
+---
+id: 8a213c
+name: First focusable element is link to main content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable element is a link to the main content of the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+  - Language
+acknowledgments:
+  authors:
+    - Christina Adams
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+There is at least one [focusable][] element within the test target.
```

Comment:
There's a disconnect between this expectation and the next. Just because there's a focusable element "within the test target", which is ambiguously phrased, doesn't mean that the sequential focus navigation order, which is a sequence of elements, is non-empty. The expectation should probably be closer to:

> The sequential focus navigation order of the test target is non-empty.